Lib/fontTools/merge.py

```diff
@@ -363,6 +364,74 @@ def _glyphsAreSame(glyphSet1, glyphSet2, glyph1, glyph2):
 		g1.width == g2.width and
 		(not hasattr(g1, 'height') or g1.height == g2.height))
 
+@_add_method(ttLib.getTableClass('CFF '))
+def merge(self, m, tables):
+	newcff = tables[0]
+	newfont = newcff.cff[0]
+	private = newfont.Private
+	storedNamesStrings = []
+	glyphOrderStrings = []
+	for name in newfont.strings.strings:
+		if name not in newfont.getGlyphOrder():
+			storedNamesStrings.append(name)
+		else:
+			glyphOrderStrings.append(name)    
+	chrset = list(newfont.charset)
+	newcs = newfont.CharStrings
+	newcsi = newcs.charStringsIndex
+	log.info("Font 0 global subrs: %s." % len(newcff.cff.GlobalSubrs))
+	ls = None
+	try:
+		ls = newfont.Private.Subrs
+	except:
+		pass
+	if ls is not None:
+		log.info("Font 0 local subrs: %s." % str(len(ls)))
```

Comment:
you don't need to call `str` to convert an integer to string if you are using the `%s` format

Reply:
and when you use logging, you don't need to evaluate the `%` formatting operators immediately, but you should do instead:

```python
log.info("Font 0 local subrs: %s", len(ls))
```

the positional argument after the first one will be interpolated only if the logging message is handled (i.e. if the client app has configured logging, and if the logger is configured for that level of verbosity, etc.)

Reply:
fixing these